Common/Models/IntentExtensionInfo.swift

```diff
@@ -0,0 +1,33 @@
+//
+//  IntentExtensionInfo.swift
+//  Loop Intent Extension
+//
+//  Created by Anna Quinlan on 10/17/20.
+//  Copyright © 2020 LoopKit Authors. All rights reserved.
+//
+
+import Foundation
+
+struct IntentExtensionInfo: RawRepresentable {
```

Comment:
I created IntentExtensionInfo so it would be easier to pass data between the intent extension and the main app in the event someone wanted to add other things to Loop's Siri capabilities